### ISSUE TITLE
Translate of ScalingCavnas

### ISSUE THREAD

**Issue by myroot** [dotnet/Microsoft.Maui.Graphics#105]
Here is implementation of `Translate` method on `ScalingCanvas`
https://github.com/dotnet/Microsoft.Maui.Graphics/blob/6e2987a1ebccfff44557f24eee9b1bf2f49e6759/src/Microsoft.Maui.Graphics/ScalingCanvas.cs#L219-L222

I am wondering it should be scaled on `Translate` method. 

It is Android NativeCanvas implementation, it also scale on `NativeTranslate`
https://github.com/dotnet/Microsoft.Maui.Graphics/blob/6e2987a1ebccfff44557f24eee9b1bf2f49e6759/src/Microsoft.Maui.Graphics/Android/NativeCanvas.cs#L778-L781

In other words, there seems to be a problem with the `Translate` of `ScalingCanvas`

it shoud be scaled
``` c#
		public void Translate(float tx, float ty)
		{
			_canvas.Translate(tx * _scaleX , ty * _scaleY);
		}
```

**Comment by myroot** [comment]
It seem to not issue
